ktxspec.adoc

```diff
@@ -671,36 +671,48 @@ source images, with the positive S axis pointing right. Where
 possible, the preferred default is to use the logical upper-left
 corner of the image as the texture origin. Note that this is contrary
 to the standard interpretation of GL texture coordinates. However,
-the majority of texture compression tools use this convention.
+most other APIs and the majority of texture compression tools use this 
+convention.
 
 As an aid to writing image manipulation tools and viewers, the
 logical orientation of the data in a KTX file may be indicated in
 the file's key/value metadata. Note that this metadata affects only
 the logical interpretation of the data, has no effect on the mapping
 from pixels in the file byte stream to texture coordinates. The
-recommended key to use is:
+metadata key must be:
 
--   KTXorientation
+-   `KTXorientation`
 
-It is recommended that viewing and editing tools support at least
-the following values:
 
--   S=r,T=d
--   S=r,T=u
--   S=r,T=d,R=i
--   S=r,T=u,R=o
+The value is a NUL-terminated string formatted depending on the texture type.
+
+[options="header"]
+|====
+|Type|Format (PCRE)
```

Comment:
PCRE stands for "Perl Compatible Regular Expressions" - loosely-defined syntax that should work with almost any environment: Boost, Java, JavaScript, Python, PHP, Ruby, Qt, .NET, XML Schema, and JSON-Schema.

Most platforms define their own "version" of PCRE (although they're very similar). Nevertheless, some platforms have external references:
- Qt points to Perl;
- JSON-Schema points to ECMA-262 standard (JavaScript).

Since the latter is probably the only fully-"standardized" and publicly-accessible option, I'd suggest us to refer to it as the reference.

Reply:
Okay. Let's use ECMA-262 but please narrow the reference to the relevant section.

Reply:
Done.